prov/efa/src/efa.h

```diff
@@ -88,6 +89,20 @@
 #define EFA_EP_TYPE_IS_DGRAM(_info) \
 	(_info && _info->ep_attr && (_info->ep_attr->type == FI_EP_DGRAM))
 
+extern struct fi_provider efa_prov;
+extern struct util_prov efa_util_prov;
+
+#define EFA_WARN(subsys, ...) FI_WARN(&efa_prov, subsys, __VA_ARGS__)
+#define EFA_TRACE(subsys, ...) FI_TRACE(&efa_prov, subsys, __VA_ARGS__)
+#define EFA_INFO(subsys, ...) FI_INFO(&efa_prov, subsys, __VA_ARGS__)
+#define EFA_INFO_ERRNO(subsys, fn, errno) \
+	EFA_INFO(subsys, fn ": %s(%d)\n", strerror(errno), errno)
+#define EFA_WARN_ERRNO(subsys, fn, errno) \
+	EFA_WARN(subsys, fn ": %s(%d)\n", strerror(errno), errno)
+#define EFA_DBG(subsys, ...) FI_DBG(&efa_prov, subsys, __VA_ARGS__)
```

Comment:
This section have been move to `efa_av.h` in previous commit, then moved back in follow up commit. Can you fix the previous commit instead?

Reply:
fixed, thanks for catching!